cdap-unit-test/src/main/java/io/cdap/cdap/test/TestBase.java

```diff
@@ -397,6 +396,32 @@ protected void configure() {
     }
   }
 
+  /**
+   * Method for enabling capability, copies the corresponding config file to appropriate location.
+   * Calling method should ensure necessary Artifact is loaded before calling
+   * Capability is enabled only on next run, so should wait for Application to be deployed
+   * and program to be running if applicable
+   *
+   * @param capability
+   * @throws IOException
+   * @throws InterruptedException
+   */
+  protected static void enableCapability(String capability) throws IOException, InterruptedException {
```

Comment:
this should be a blocking call that returns once the capability is enabled, otherwise it will be difficult to use. We should be able to get the right class through the injector and call whatever method is used to enable/add a capability directly instead of going through this file pickup process.

Reply:
As per our discussion, calling CapabilityManagementService runTask to speed up enable/disable